sandbox/grommet-app/src/themes/theme.jsx

```diff
@@ -868,15 +871,16 @@ const buildTheme = tokens => {
           large: elevationdark
             ? elevationdark.hpe.elevation.large
             : dark.hpe.shadow.large,
+          'inset-selected': `inset 3px 0 ${dark.hpe.color.border.selected}`,
```

Comment:
Do you see the `3px` be replaced by a spacing token in the future?

Reply:
Yes, if we end up going this "bookend" approach, I would imagine we'd add all parts of this style as a token.